### DISCUSSION TITLE
search results order

### DISCUSSION THREAD

**Discussion by lialang39** [sheltermanager/asm3#1494] [Q&A / Help]
Is there a way to make the search results in a specific order? We have cats names that have been used so many times that when we search for the cat, for example "Socks", we have to page down looking for the most recent cat. If the results could be in order of newest to oldest, that would be most helpful.

**Comment by bobintetley** [comment]
Yes, you can choose the default order under Settings->Options->Search. There's a "Most Recently Changed" option. You can also do it just for the search you are running by adding sort:mr to the end of your search query - eg: "socks sort:mr"
